### PR TITLE
feat: updated the interface for the latest sdk version

### PR DESCRIPTION
**Pull request by GabrielBottari**

### CODE REVIEW

**Review by chammaaomar — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "com.mistplay.loyaltyplay",
-    "version": "1.0.2",
+    "version": "1.1.0",
```
Comment (chammaaomar) [inline package.json:3]: That's weird I thought `package.json` is only for node projects

File: Runtime/LoyaltyPlay.cs [hunk 2]
```diff
@@ -57,23 +57,29 @@ private void SetLanguageInternal(string language)
         /// Launches Loyalty Play.
         /// (The userId is the unique id for each user to associate to a Loyalty Play profile)
         /// </summary>
-        /// <param name="userId"></param>
-        public static async Task Launch(string userId)
+        /// <param name="userId">The user id.</param>
+        /// <param name="startingCurrencyBalance">The current user currency balance to build upon. Solely used for UX purposes. Defaults to the total currency gained.</param>
+        /// <param name="showSessionEndDialog">Show a Rewards Summary dialog on LoyaltyPlay close. Can be overriden with a custom dialog by turning this off and using the sessionEndCallback. Defaults to true.</param>
+        /// <param name="sessionEndAction">An action that will be fired on LoyaltyPlay close, with the user id and currency gained for that session as parameters.</param>
```
Comment (chammaaomar) [inline Runtime/LoyaltyPlay.cs:63]: Are these messages user-facing? By user I mean client dev using the SDK.

Comment (GabrielBottari) [reply Runtime/LoyaltyPlay.cs:63]: Yes for devs, but they aren't showing except for the summary so not sure 🤔

File: package.json (hunk 1 above)
Comment (GabrielBottari) [reply package.json:3]: Yeah.. thats what unity uses

**Pull request merged**